Fixing max_flow ordering issue

krashish8 opened the pull request:
max_flow on #68.

Changes proposed in this pull request:
- Adds pgTAP tests for max_flow directory that check the output after rows ordering.

@pgRouting/admins

krashish8 commented:
The pgTAP tests fail. So, fixing the code.

cvvergara commented on include/max_flow/pgr_maxflow.hpp:
```diff
@@ -142,12 +142,14 @@ class PgrFlowGraph {
      void set_supersink(
              const std::set<int64_t> &sink_vertices);
 
+     void sort_edges(
```
Whatever you are doing as a fix its not working:
https://github.com/pgRouting/GSoC-pgRouting/runs/899643812#step:6:114

Its not building, so:
return the code to what it was and wrap the failing tests with todo.

This is not a quick fix so you are not doing that fix (at least not during the GSoC program)

krashish8 commented:
I have reverted the changes and added the todo.
However, the build and tests seem to pass: https://github.com/pgRouting/GSoC-pgRouting/runs/901211331

Initially, it was failing, but on merging branch 'demo-master' into issue-68-max_flow, the checks passed.
